lib/assets/javascripts/dashboard/dashboard.js

```diff
@@ -196,3 +199,12 @@ if (window.CartoConfig.data) {
     }
   });
 }
+
+function resetOrderWhenLikesIsApplied () {
+  const localStorageInstance = new LocalStorage();
+  const dashboardOrder = localStorageInstance.get('dashboard.order');
+
+  if (dashboardOrder === 'likes') {
+    localStorageInstance.set({'dashboard.order': 'mapviews'});
```

Comment:
Default order is by date, should we change it to that?

```suggestion
    localStorageInstance.set({'dashboard.order': 'updated_at'});
```

Reply:
Yes, sure thing 😓